use-cases/inferencing/serving/vllm/autoscaling/README.md

```diff
@@ -123,13 +122,13 @@ Service for Prometheus [documentation](https://cloud.google.com/kubernetes-engin
   - Queue-depth
 
     ```sh
-    kubectl --namespace ${SERVE_NAMESPACE} apply -f manifests/hpa-vllm-openai-queue-size.yaml
+    kubectl --namespace ${MLP_MODEL_OPS_NAMESPACE} apply -f manifests/hpa-vllm-openai-queue-size.yaml
```

Comment:
I think these should be `MLP_MODEL_SERVE_NAMESPACE` as that is where the model server is deployed